CRM/Mailing/BAO/Mailing.php

```diff
@@ -2519,8 +2523,8 @@ public static function del($id) {
   }
 
   /**
-   * Delete Jobss and all its associated records
-   * related to test Mailings
+   * @deprecated
+   *  Use CRM_Mailing_BAO_MailingJob::del($id)
```

Comment:
If you pull this delete part into a PR I should be able to merge that pretty quickly

Reply:
Done, extracted del codes in separate PR https://github.com/civicrm/civicrm-core/pull/12275

Reply:
ok -that is merged - we can rebase this